tests/acceptance/17_packages/10_new/unsafe/packagesmatching.cf

```diff
@@ -0,0 +1,55 @@
+# Test that updates list cache is not updated too often.
+
+body common control
+{
+    inputs => { "../../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+
+    package_inventory => { test_module };
+    #package_module => "test_module";
```

Comment:
I was not entirely sure if we will need this one or not. Looks like we can remove it.